cmd/protoc-gen-gorums/dev/server.go

```diff
@@ -61,33 +47,14 @@ func (s *orderingServer) NodeStream(srv ordering.Gorums_NodeStreamServer) error
 		}
```

Comment:
The go func() above may exit if `srv.SendMsg()` return an `err`. Would this error message be relevant to return from the `NodeStream` func. Or alternatively, it would be good to have a comment that explain why it's okay to exit this goroutine without exiting from the `NodeStream` func.

Reply:
If either `SendMsg()` or `RecvMsg()` return an `error` then the stream is aborted. So, if an error happens in the sending goroutine, then the receiving goroutine should also get an error and vice-versa.